modules/oe_whitelabel_search/tests/src/Kernel/SearchBlockTest.php

```diff
@@ -84,7 +82,7 @@ protected function setUp(): void {
   }
 
   /**
-   * Tests the rendering of blocks.
+   * Tests the rendering of the whitelabel search block.
    */
   public function testBlockRendering(): void {
     $entity_type_manager = $this->container
```

Comment:
The variable name should be `$block_entity_storage`, not `$entity_type_manager`.
(Up to you if you want to fix it in this PR, because it was not introduced here)